geotrek/signage/tests/test_parsers.py

```diff
@@ -67,3 +73,77 @@ def test_create(self, mocked_head, mocked_get):
         self.assertEqual(conditions, ["Dégradé"])
         self.assertAlmostEqual(signage.geom.x, 572941.1308660918, places=5)
         self.assertAlmostEqual(signage.geom.y, 6189000.155980503, places=5)
+
+
+class TestSignageOpenStreetMapParser(OpenStreetMapSignageParser):
+    provider = "OpenStreetMap"
+    tags = [{"information": "board"}]
+    default_fields_values = {"name": "Test"}
+    type = "Test"
+
+
+class OpenStreetMapSignageParser(TestCase):
+    @classmethod
+    @mock.patch("geotrek.common.parsers.requests.get")
+    def import_Signage(cls, mocked):
+        def mocked_json():
+            filename = os.path.join(
+                os.path.dirname(__file__), "data", "signage_OSM.json"
+            )
+            with open(filename) as f:
+                return json.load(f)
+
+        mocked.return_value.status_code = 200
+        mocked.return_value.json = mocked_json
+
+        call_command(
+            "import",
+            "geotrek.signage.tests.test_parsers.TestSignageOpenStreetMapParser",
+        )
+
+    @classmethod
+    def setUpTestData(cls):
+        cls.type = SignageType.objects.create(label="Test")
+        cls.path = PathFactory.create(
+            geom=LineString((6.3737905, 44.8337000), (6.3757905, 44.8357000), srid=4326)
+        )
+        cls.import_Signage()
+        cls.objects = Signage.objects.all()
+
+    @skipIf(
+        not settings.TREKKING_TOPOLOGY_ENABLED, "Test with dynamic segmentation only"
+    )
+    def test_import_cmd_raises_error_when_no_path(self):
+        self.path.delete()
+        with self.assertRaisesRegex(
+            CommandError, "You need to add a network of paths before importing Signages"
+        ):
+            call_command(
+                "import",
+                "geotrek.signage.tests.test_parsers.TestSignageOpenStreetMapParser",
+                verbosity=0,
+            )
+
+    def test_create_Signage_OSM(self):
+        self.assertEqual(self.objects.count(), 1)
+
+    def test_Signage_eid_filter_OSM(self):
+        signage_eid = self.objects.order_by("eid").all().values_list("eid", flat=True)
```

Comment:
Since it's a queryset, I'd add an 's' even though in this case it only contains one element. In the future, more elements could be added.

```suggestion
        signage_eids = self.objects.order_by("eid").all().values_list("eid", flat=True)
```